samples/MvcSample.Web/Startup.cs

```diff
@@ -63,12 +63,6 @@ public IServiceProvider ConfigureServices(IServiceCollection services)
                 diSystem.Equals("AutoFac", StringComparison.OrdinalIgnoreCase))
             {
                 _autoFac = true;
-                services.ConfigureRazorViewEngine(options =>
-                {
-                    var expander = new LanguageViewLocationExpander(
-                        context => context.HttpContext.Request.Query["language"]);
-                    options.ViewLocationExpanders.Insert(0, expander);
-                });
```

Comment:
There should still be some kind of equivalent gesture for enabling this feature or at least for enabling MVC localization features.